tests/Microsoft.Identity.Test.Unit/WwwAuthenticateParametersTests.cs

```diff
@@ -156,6 +157,14 @@ public async Task CreateFromResourceResponseAsync_HttpClient_Arm_GetTenantId_Asy
             var authParams = await WwwAuthenticateParameters.CreateFromResourceResponseAsync(httpClient, resourceUri).ConfigureAwait(false);
 
             Assert.AreEqual(authParams.GetTenantId(), tenantId);
+

```

Comment:
THere doesn't seem to be enough unit tests: 

- claims in base64url format / json format / bad format
- more parsing errors
- extra auth schemes.